--- glcTransformFeedbackTests.cpp	2024-07-25 21:39:20.000000000 +0700
+++ VK-GL-CTS-opengl-es-cts-3.2.11.0/external/openglcts/modules/common/glcTransformFeedbackTests.cpp	2024-12-09 14:41:16.151571231 +0700
@@ -438,7 +438,12 @@
         GLU_EXPECT_NO_ERROR(gl.getError(), "getQueryObjectuiv");
     }
 
-    gl.getQueryObjectuiv(m_queries[1], GL_QUERY_RESULT, &queryresults[1]);
+    // gl.getQueryObjectuiv(m_queries[1], GL_QUERY_RESULT, &queryresults[1]);
+    GLuint available = 0;
+    gl.getQueryObjectuiv(m_queries[1], GL_QUERY_RESULT_AVAILABLE, &available);
+    
+    if(available != 0)
+        gl.getQueryObjectuiv(m_queries[1], GL_QUERY_RESULT, &queryresults[1]);
     GLU_EXPECT_NO_ERROR(gl.getError(), "getQueryObjectuiv");
 
     if ((!m_isContextES && queryresults[0] != 5) || queryresults[1] != 2)
